docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -58,6 +58,7 @@
 |spring.cloud.gateway.filter.set-status.enabled | `true` | Enables the set-status filter.
 |spring.cloud.gateway.filter.strip-prefix.enabled | `true` | Enables the strip-prefix filter.
 |spring.cloud.gateway.forwarded.enabled | `true` | Enables the ForwardedHeadersFilter.
+|spring.cloud.gateway.forwarded.by-enabled | `true` | If appending Forwarded By header.
```

Comment:
If I'm not mistaken the configuration parameter should be named:
`spring.cloud.gateway.forwarded.by.enabled`

We could also remove this configuration. I don't know why you wouldn't want to have the `by` attribute if you have the forwarded header enabled.

Reply:
Good point! As I understand [RFC 7239](https://datatracker.ietf.org/doc/html/rfc7239#section-5.1) the default configuration for "Forwarded By" should be an obfuscated identifier. It may be also contain an ip address.  A third option is the "unknown" identifier. 

I could update this PR towards using "_spring-cloud-gateway" as a default (obfuscated) identifier, make ip address configurable and "unknown" as fallback, if ip address is not known to the proxy.